supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -8,7 +8,7 @@
 Follow these guidelines when adding cross-references within your documentation:
 
 * Include cross-references only when necessary.
-* If the information is critical, consider including it instead of cross-referencing.
+* When not using running text,, consider including it instead of cross-referencing.
```

Comment:
So the line to modify for this one was line 61, if you can make the adjustment there instead

Reply:
Done @bergerhoffer